### PR TITLE
@redyaffle Please look this over. I'm going to add tests to it before submitting it for real.

### PR DESCRIPTION
**Pull request by bantic**
_do not merge_

### CODE REVIEW

**Comment by bantic** [comment]
The ember-side PR is https://github.com/201-created/LG/pull/4

**Comment by bantic** [comment]
does not do any authorization of requests. that will happen in a separate PR. This just signs in a user  via the sessions controller and then signs that user in via token for all api requests.

File: db/migrate/20150127022807_create_image_types.rb [hunk 1]
```diff
@@ -8,9 +8,5 @@ def change
       t.index :name, :unique => true
     end
 
-    ['CV', 'Whisker', 'Main ID', 'Full Body'].each do |image_type|
-      ImageType.create(:name => image_type.downcase.gsub(" ", "-"),
-                       :display_name => image_type)
-    end
```
Comment (bantic) [inline db/migrate/20150127022807_create_image_types.rb:14]: this breaks up migrations because `ImageType` doesn't exist anymore

**Comment by bantic** [comment]
@redyaffle this can be merged now.

**Comment by redyaffle** [comment]
@bantic Looks good. Cool to see some of the tricks in the tests you implemented.

**Pull request merged**